### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by jessedobbelaere**
Should fix (hopefully) all broken internal links

### CODE REVIEW

File: 06. contribute/02. testing.md [hunk 1]
```diff
@@ -23,6 +23,3 @@ Best practices considering security can be reported in the documentation, submit
 ## Automated testing
 
 (TODO)
-
-
-While testing is one of the most valueable contributions, it doesn't allow you to build your creative ideas into reality. The next article is about [adding new features](features).
```
Comment (carakas) [inline 06. contribute/02. testing.md]: why was this removed?

File: 06. contribute/01. introduction.md [hunk 2]
```diff
@@ -87,12 +87,12 @@ Not all decisions can be made using lazy consensus. Issues such as those affecti
...
-* [Spreading the word](spreading-the-word)
+* [Testing](02. testing.md)
+* [Features](07. features.md)
+* [Translations](04. translations.md)
+* [User Interface](05. user_interface.md)
+* [Documentation](06. documentation.md)
+* [Spreading the word](06. spreading_the_word.md)
 
 
-*This document is based on the [Template For A Meritocratic Governance Document](http://www.oss-watch.ac.uk/resources/meritocraticGovernanceModel) of OSS Watch*
```
Comment (carakas) [inline 06. contribute/01. introduction.md]: what has changed here? I can't spot i

File: 06. contribute/07. features.md [hunk 3]
```diff
@@ -25,6 +25,3 @@ After you installed [Git](https://help.github.com/articles/set-up-git), start by
 After you [comitted](https://help.github.com/articles/create-a-repo#step-2-commit-your-readme) all changes to your branch and [pushed](https://help.github.com/articles/create-a-repo#step-3-push-your-commit) them to your forked version, you'll have to do a [pull request](https://help.github.com/articles/using-pull-requests). Make sure you explain what and why you want to change a part to the core. After the pull request has been voted or discussed on the [technical board meeting](index.md) it will get merged.
 
 More information about working with GitHub can found on the [help](https://help.github.com/) of GitHub, in the [documentation](http://git-scm.com/documentation) of Git, in the [Git Workflows](http://documentup.com/skwp/git-workflows-book) book or learn Git Branching [interactively](http://pcottle.github.io/learnGitBranching/).
-
-
-When you edit something in the core we will ask you to update the documentation too. Read our [documentation](03. documentation.md) article to find some tips.
```
Comment (carakas) [inline 06. contribute/07. features.md]: well, I think it would be a good idea to start doing this again instead of removing this

File: 06. contribute/02. testing.md (hunk 1 above)
Comment (jessedobbelaere) [inline 06. contribute/02. testing.md]: Because for some reason the "features" article is the 07'th article. Should I write some text to announce the third article "documentation"?

Btw I think the whole contribute folder is too bloated and can be summarized into one article "Contributing"

File: 06. contribute/01. introduction.md (hunk 2 above)
Comment (jessedobbelaere) [inline 06. contribute/01. introduction.md]: Strange, nothing changed. I used the online github edit functionality (easier to preview the markdown), so could be because of that. Ignore ;)

File: 06. contribute/07. features.md (hunk 3 above)
Comment (jessedobbelaere) [inline 06. contribute/07. features.md]: Re-added with correct order of files and links

File: 06. contribute/01. introduction.md (hunk 2 above)
Comment (jessedobbelaere) [inline 06. contribute/01. introduction.md]: Fixed the order of documents and re-added this line
